In [12]:

name ='Rehan'
hourly_wage=50.5
hour=5
weekly_pay= hourly_wage * hour
print(f'{name} earns {weekly_pay} per week')



Rehan earns 252.5 per week


In [13]:
temps=[61,65,72,58,70,74,68]
count = 0
for i in temps:
    if i > 65:
        count = count+1
print (count)        
        

4


In [18]:
def to_celsius(f):
    return (f-32)*5/9
print(round(to_celsius(temps[0]),1))    

16.1


In [20]:
movie_dic={'title': 'Hard Homework' ,'year': 2004,'rating':5}
for key,value in movie_dic.items():
    print(key, ":", value)

title : Hard Homework
year : 2004
rating : 5


In [36]:
import pandas as pd
df = pd.read_csv(r"C:\Users\rehan\Downloads\chickwts (1).csv")
print(df.head())
print(df.shape)
df.describe()

   Unnamed: 0  weight       feed
0           1     179  horsebean
1           2     160  horsebean
2           3     136  horsebean
3           4     227  horsebean
4           5     217  horsebean
(71, 3)


,Unnamed: 0,weight
count,71.000000,71.000000
mean,36.000000,261.309859
std,20.639767,78.073700
min,1.000000,108.000000
25%,18.500000,204.500000
50%,36.000000,258.000000
75%,53.500000,323.500000
max,71.000000,423.000000


The number of chicken are 71. The average weight is 261. The suprising thing is i don't now what is the unnamed column is. 